core/tests/protractor.conf.js

```diff
@@ -319,9 +323,52 @@ exports.config = {
     // will be available. For example, you can add a Jasmine reporter with:
     //     jasmine.getEnv().addReporter(new jasmine.JUnitXmlReporter(
     //         'outputdir/', true, true));
-    var _ADD_SCREENSHOT_REPORTER = true;
 
-    if (_ADD_SCREENSHOT_REPORTER) {
+    var spw = '';
+    var uniqueString = '';
+    var vidPath = '';
+    // Enable allVideos if you want success videos to be saved.
+    var allVideos = false;
```

Comment:
If this is a constant, can we make it all-caps?

Reply:
done